Dockerfile

```diff
@@ -55,6 +52,7 @@ RUN set -ex \
     && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 \
     && useradd -ms /bin/bash -d ${AIRFLOW_HOME} airflow \
     && pip install -U pip setuptools wheel \
+    && pip install requests \
```

Comment:
requests is already installed via Airflow setup https://github.com/apache/incubator-airflow/blob/master/setup.py#L323